ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -23,8 +23,15 @@ export interface TimeSeriesChartOptions {
   legend?: LegendOptions;
   unit?: UnitOptions;
   thresholds?: ThresholdOptions;
+  visual?: VisualOptions;
 }
 
+export type VisualOptions = {
+  // type: 'line' | 'bar' | 'scatter'; // TODO: new option to change series type
```

Comment:
Ability to customize series type (switch to bar and scatterplot similar to Grafana's custom.drawStyle property) will come in a future PR, I left this commented line as an example in case anyone has thoughts / questions

Reply:
@Nexucis to your comment about sharing `visual` outside of panels/time-series/time-series.cue, my plan is to add this `visual.type` option in the future to switch the series type to 'bar' and 'scatter'. This would be implemented in `kind: 'TimeSeriesChart'`, so at that point I don't see this version of 'visual' being reused anywhere else (other future panels will have their own visual options though)